CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 #### ⚠️ Pushed to `master`
 
+- Use HTTP GET for `stream` requests in @gitbeaker/node ([@Vogel612](https://github.com/Vogel612))
```

Comment:
```diff
- - Use HTTP GET for `stream` requests in @gitbeaker/node ([@Vogel612](https://github.com/Vogel612))
```

unneeded because changelog is autogenerated